File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.py
def solution(order):
    # method1
    # s=0
    # for o in order:
    #     if o.startswith('ice') or o.endswith('ice'):
    #         s+=4500 if 'ame' in o else 5000
    #     elif o.startswith('hot') or o.endswith('hot'):
    #         s+=4500 if 'ame' in o else 5000
    #     elif o.startswith('ame') or o.startswith('any'):
    #         s+=4500
    #     else:
    #         s+=5000
    # method2
    s=sum([5000 if 'latte' in o else 4500 for o in order])
    return s

File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// order_len은 배열 order의 길이입니다.
// 파라미터로 주어지는 문자열은 const로 주어집니다. 변경하려면 문자열을 복사해서 사용하세요.
int solution(const char* order[], size_t order_len) {
    int r=0;
    for(int i=0;i<order_len;i++)
        r+=strstr(order[i],"latte")!=0?5000:4500;
    return r;
}